tools/commands/deploy.py

```diff
@@ -107,33 +124,39 @@ def mkdir_p(path):
         else:
             raise
 
+
 def reset(tarinfo):
     tarinfo.uid = tarinfo.gid = 0
     tarinfo.uname = tarinfo.gname = "root"
     return tarinfo
 
+
 class mapdeploy(object):
     def __init__(self, args):
-        if args.verbose: logger.setLevel(logging.DEBUG)
-        modules = deploy_modules if 'all' in args.modules else [m for m in deploy_modules if m in args.modules]
+        if args.verbose:
+            logger.setLevel(logging.DEBUG)
+        modules = deploy_modules if 'all' in args.modules else [
+            m for m in deploy_modules if m in args.modules]
```

Comment:
Meh, it's a bit silly to fix this since it actually doesn't work any more... There are no longer any modules...

Oh well, let's just fix it for pep8.

Reply:
Fixed for PEP8